prow/prstatus/prstatus.go

```diff
@@ -168,20 +169,44 @@ func (da *DashboardAgent) HandlePrStatus(queryHandler PullRequestQueryHandler) h
 		data := UserData{
 			Login: false,
 		}
+
+		login := ""
 		if ok && token.Valid() {
-			data.Login = true
-			ghc := github.NewClient(token.AccessToken, githubEndpoint)
-			// Get user login
+			// If access token exist, get user login using the access token. This is a chance
+			// to validate whether the access token is consumable or not. If not, invalidate the
+			// session.
```

Comment:
The `getUserLogin` function doesn't necessarily get the user login using the access token. It might just use the stored session value in which case validation is skipped?

Reply:
I changed it to request for user login everytime